spec/Section 2 -- Language.md

```diff
@@ -815,27 +826,66 @@ StringCharacter ::
   - `\u` EscapedUnicode
   - `\` EscapedCharacter
 
-EscapedUnicode :: /[0-9A-Fa-f]{4}/
+EscapedUnicode ::
+  - `{` HexDigit+ `}`
+  - HexDigit HexDigit HexDigit HexDigit
+
+HexDigit :: one of
+  - `0` `1` `2` `3` `4` `5` `6` `7` `8` `9`
+  - `A` `B` `C` `D` `E` `F`
+  - `a` `b` `c` `d` `e` `f`
 
 EscapedCharacter :: one of `"` `\` `/` `b` `f` `n` `r` `t`
 
 BlockStringCharacter ::
   - SourceCharacter but not `"""` or `\"""`
   - `\"""`
 
-Strings are sequences of characters wrapped in quotation marks (U+0022).
-(ex. {`"Hello World"`}). White space and other otherwise-ignored characters are
-significant within a string value.
+{StringValue} is a sequence of characters wrapped in quotation marks (U+0022).
+(ex. {`"Hello World"`}). White space and other characters ignored in other parts
+of a GraphQL document are significant within a string value.
+
+A {StringValue} is evaluated to a Unicode text value, a sequence of Unicode
+scalar values, using the semantics defined below.
```

Comment:
@andimarek this is based on your suggestion to be explicit that the grammar source and the encoded string value are not the same

Reply:
@leebyron can we make clear what evaluate means: unescaping the literal by interpreting the escape sequences?